bot/utils/helpers.py

```diff
@@ -30,3 +31,14 @@ def has_lines(string: str, count: int) -> bool:
 def pad_base64(data: str) -> str:
     """Return base64 `data` with padding characters to ensure its length is a multiple of 4."""
     return data + "=" * (-len(data) % 4)
+
+
+def join_role_stats(role_ids: List[int], name: str, guild: Guild) -> Dict[str, int]:
```

Comment:
Like @jchristgit mentioned this is not really useful for other code so I don't think it fits into this module as a general util. I think having it in the information module or cog directly would make a bit more sense.

The bot is also running on python 3.9 so the generics f rom typing aren't required and builtins can be used directly